public/pull_request_template

```diff
@@ -0,0 +1,20 @@
+## Describe your changes
+
+## Issue ticket number and link
```

Comment:
Maybe let's remove this section and add a `Issue=#<Number>` at the top of the template?

For instance for this PR:

```
Issue=#5
```